app/controllers/users.js

```diff
@@ -47,3 +48,26 @@ exports.getUsers = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.createAdminUser = async (req, res, next) => {
+  try {
```

Comment:
For these cases we create interactors https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#26--interactors